scripts/beegfspkgs.sh

```diff
@@ -11,7 +13,13 @@ yum install -y kernel-devel-${KERNEL} kernel-headers-${KERNEL} kernel-tools-libs
 yum -y install gcc gcc-c++
 yum -y install zlib zlib-devel bzip2 bzip2-devel bzip2-libs openssl openssl-devel openssl-libs nfs-utils rpcbind mdadm wget python-pip automake autoconf
 #
+if [ ${BEEGFS_VER} -eq 713 ]
```

Comment:
You may test if BEEGFS_VER contains a value and in that case use that variable in the wget and the sed so that your script will work for any other versions